charts/fluid/fluid/values.yaml

```diff
@@ -9,6 +9,8 @@ crdUpgrade:
 image:
   imagePullSecrets: []
 
+namespace:
```

Comment:
how about set `fluid-system` as default value to maintain previous behavior and prevent conflicts when upgrading?

Reply:
-n fluid-system

Reply:
I agree with @zwwhdls because we'd better not introduce any installation or upgrade change for the existing users. I think they will use `fluid-system` namespace if not specifying anything.

Reply:
`helm install -n fluid-system` will change helm release from `default` to `fluid-system`. It's not make sense with existing users.